controllers/reviews/getReviewById/getReviewById.js

```diff
@@ -0,0 +1,13 @@
+import { Review } from '../../../models/review/index.js';
+
+export const getReviewById = async (app) => {
+  app.get('/reviews/:reviewId', async (req, res, next) => {
```

Comment:
do we need to request just one review?

Reply:
I think no.
I made it, but now I can't find cases where we can use it.
It was in Miro. I discuss with my team this question.